### PR TITLE
New perma-id for new ontology "ESIM"

### PR DESCRIPTION
**Pull request by HervePruvost**
Hello,
I would like to register a new perma-id URL for my newly added folder ./esim.
README.md and .htaccess files are ready and included in the esim folder.
Everything seems valid.

Thank you in advance
BR
Hervé Pruvost
(Fraunhofer IIS/EAS)

### CODE REVIEW

File: esim/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Energy System Information Model (ESIM)
+
+This [W3ID](https://w3id.org) provides a persistent URI namespace for the ESIM (Energy System Information Model) ontology.
+
+## Uses
+
+ESIM (Energy System Information Model) is a domain-specific metadata model that is represented as an RDF graph 
+and that describes information about energy systems at building and urban levels including their automation and
+control systems. It comprises functional, structural and physical descriptions of the
+systems as master metadata and additional operational metadata.
```
Comment (TallTed) [inline esim/README.md:10]:
```suggestion
ESIM (Energy System Information Model) is a domain-specific metadata model, is represented as an RDF graph,
that describes information about energy systems at building and urban levels including their automation and
control systems. It comprises functional, structural, and physical descriptions of the
systems as master metadata and additional operational metadata.
```

File: esim/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+This space is administered by:
+
+**Hervé Pruvost**  
+_R&D Engineer_  
+[Fraunhofer IIS/EAS](hhttps://www.eas.iis.fraunhofer.de/)  
+Dresden, Germany  
+<herve.pruvost@eas.iis.fraunhofer.de>  
+GitHub: [HervePruvost](https://github.com/HervePruvost)
+Linkedin: [Hervé Pruvost](https://www.linkedin.com/in/hervepruvost/)
+ORCID: [0000-0001-7604-8543](https://orcid.org/0000-0001-7604-8543)
```
Comment (TallTed) [inline esim/README.md:23]: to maintain line breaks when markdown is rendered
```suggestion
<herve.pruvost@eas.iis.fraunhofer.de>  
GitHub: [HervePruvost](https://github.com/HervePruvost)  
Linkedin: [Hervé Pruvost](https://www.linkedin.com/in/hervepruvost/)  
ORCID: [0000-0001-7604-8543](https://orcid.org/0000-0001-7604-8543)
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Small stuff

File: esim/README.md [hunk 3]
```diff
@@ -4,9 +4,9 @@ This [W3ID](https://w3id.org) provides a persistent URI namespace for the ESIM (
 
 ## Uses
 
-ESIM (Energy System Information Model) is a domain-specific metadata model that is represented as an RDF graph 
-and that describes information about energy systems at building and urban levels including their automation and
-control systems. It comprises functional, structural and physical descriptions of the
+ESIM (Energy System Information Model) is a domain-specific metadata model, is represented as an RDF graph,
```
Comment (TallTed) [inline esim/README.md:7]: I really did mean for the "is" in "is represented" to be deleted... for the comma after "model" to replace "that is".

Comment (HervePruvost) [reply esim/README.md:7]: You are right. I made a small PR #3548 to correct this.

**Pull request merged**